### ISSUE TITLE
Orthology core genes question

### ISSUE THREAD

**Issue by FreBaert** [Lcornet/GENERA#15]
**Which GENERA tool**
Give here the name of the tool:
Orthology
**More about**
Explain below what is unclear:
When I try to run the Orthology tool with the core genes option I get the following error. I suspect the issue is that none of the OGs fit the criteria for being identified as a core gene (100% of presence with no duplication in of OGs are the criteria used to determine the core genes), is there any way for me to verify that this is the case?

```
executor >  local (3)
[87/bac473] process > format (1)         [100%] 1 of 1  ^|^t
[bb/6bf981] process > orthofinder (1)    [100%] 1 of 1  ^|^t
[ca/4ca91f] process > core (1)           [  0%] 0 of 1
[-        ] process > publicationResults -

Error executing process > 'core (1)'

Caused by:
  Missing output file(s) `CORE/OG*.fa` expected by process `core (1)`

Command executed:

  #mkdir OG
  mkdir CORE
  #mv *.fa OG
  cd OG
  /opt/COMPANION/Orthology_companion.py ../list
  cd ../
  mv OG/core-OG.list .
  sed -i -e 's/.fa//g' core-OG.list
  for f in `cat core-OG.list`; do mv OG/$f.fa CORE/; done
  echo "GENERA info: core gene option activated" >> GENERA-Orthology.log
  echo "Number of Core gene:" >> GENERA-Orthology.log
  wc -l core-OG.list >> GENERA-Orthology.log

Command exit status:
  0

Command output:
  (empty)

Work dir:
  /scratch/users/f/b/fbaert/test/ortho_core/work/ca/4ca91f4133aba681311b678a3e0340
```

GENERA-orthology.log:

```
GENERA info: format names with BMC
GENERA info: run Orthofinder
GENERA info: core gene option activated
Number of Core gene:
0 core-OG.list

```

**Comment by Lcornet** [comment]
I think it is due to the format of the proteomes deflines:

>sp|A0A059J5P2|DEFTR_TRIIM. 
-> It's already abbreviate which is a problem for the workflow.

I add a section about input files in the doc:
https://github.com/Lcornet/GENERA/wiki/09.-Orthology

Fell free to reopen if it's still unclear.

**Comment by FreBaert** [comment]
I ran this analysis without the core genes option with the same inputs and it was fine, so I think there must be another issue? 
I can see that in my run without core genes enabled the deflines were changed from f.e. :
`tr|A0A059JD68|A0A059JD68_TRIIM Uncharacterized protein OS=Trichophyton interdigitale (strain MR816) OX=1215338 GN=H109_02346 PE=4 SV=1`
to
`>uniprot-proteome_UP000024533.interdigitale|tr|A0A059JD68|A0A059JD68_TRIIM`

'uniprot-proteome_UP000024533.interdigitale.faa' was indeed the name of the input file so it seems to be working correctly?

I tried running the analysis without any '|' symbols in the input deflines and I am getting the same error as before.
something like:
```
>interdigitale|tr_A0A059JKQ0_A0A059JKQ0_TRIIM
MPLVVPNVSEDDKAAWAAKLMGKKITESESNETCFAKKDLPEGHRVLKPGDMSTMDFKPE
RMNVYVDENGTVMDVKHG
>rubrum|tr_F2T0T4_F2T0T4_TRIRC
MPLVVPNLSEDDKAAWAAKLMGKKITESESNETCFAKKDLPEGHRVLKPGDMSTMDFKPE
RMNVYVDENGTVMDVKHG
>verrucosum|augustus-GCF_000151505.1_NW_003315554.1-abinit-gene-0.14-mRNA-1
MTSISRGAPDESFIKTIVKTAHHLKKLLAPGRFTSTSSTPSTMPLVVPNLSEDDKAAWAA
KLMGKKITESESNETVSLSPLMHAGYSGTDIAFLPCFLVFCKEGPSRGTPSSQAR
```
 
should be an acceptable input for core genes proces right?
